convene-web/config/routes.rb

```diff
@@ -1,26 +1,29 @@
-
-
 Rails.application.routes.draw do
   passwordless_for :people
 
   namespace :admin do
-      resources :workspaces
-      resources :clients
-      resources :people
-      resources :workspace_memberships
-      resources :room_ownerships
-      resources :rooms
+    resources :workspaces
+    resources :clients
+    resources :people
+    resources :workspace_memberships
+    resources :room_ownerships
+    resources :rooms
+
+    root to: 'workspaces#index'
+  end
 
-      root to: "workspaces#index"
-    end
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   resources :workspaces do
-    resources :rooms, only: [:show, :edit, :update] do
-      resource :waiting_room, only: [:show, :update]
+    resources :rooms, only: %i[show edit update] do
+      resource :waiting_room, only: %i[show update]
+      namespace :furniture do
+        resources :tables, only: [:show], controller: 'breakout_tables/'
```

Comment:
[Non-blocking] There is no show action in `BreakoutTables`, I assume this is intentional to be an example on how to add routes to point to this controller?